### ISSUE TITLE
Incorrect formula in docstring for torch.nn.modules.normalization.RMSNorm

### ISSUE THREAD

**Issue by enedil** [pytorch/pytorch#144659]
### 📚 The doc issue

Issue is at this line:
https://github.com/pytorch/pytorch/blob/9ae35b8bb13f3c35803355ce26fb9ee9954f1bdf/torch/nn/modules/normalization.py#L328

Assume for simplicity, that len(x.shape) == 1.

Formula is 
```
y = x / sqrt(RMS[x] + epsilon) * gamma
```

RMS, as indicated from the linked ARXIV paper, sqrt(mean(x^2)). 
So, according to the docstring, this formula is correct:
```
y = x / sqrt(sqrt(mean(x^2)) + epsilon) * gamma
```

RMSNorm computes however something else, namely
```
y = x / sqrt(mean(x^2) + epsilon) * gamma
```

This is apparently what RMSNorm should do, so the issue is in the docs.

### Suggest a potential alternative/fix

Change the formula not to refer to RMS[x], if we want epsilon to be included, or introduce a term like RMS[x, eps]? It is not clear for me how to make this legible and compact.

**Comment by YouJiacheng** [comment]
where does the `sqrt` come from?
It just `y = (x / RMS(x)) * γ`

**Comment by enedil** [comment]
Citing my original issue statement:
> RMS, as indicated from the linked ARXIV paper, sqrt(mean(x^2)).

Do you mean by RMS something else, namely just mean(x^2)?

**Comment by YouJiacheng** [comment]
Ah I got it. You cited a permanent link (i.e. with commit hash) in this issue but that permanent link is actually without the outer `sqrt`.
But true, the stable version docs is wrong.
